projects/commudle-admin/src/app/components/home/components/home-external-feed-links/home-external-feed-links.component.scss

```diff
@@ -1,16 +1,85 @@
 @import '/projects/commudle-admin/src/assets/styles/variables';
 
-.external-feed-links {
-  border: none;
+nb-card-header, div, a, p {
+  font-family: $primary-font-family;
+}
+
+.home-external-feed-links {
   border-radius: $border-10;
 
-  a {
-    text-decoration: none;
+  nb-card-header {
+    font-size: $fs-h2;
+    font-weight: $fw-600;
     color: $gunmetal;
-    font-weight: $fw-500;
+    padding: $space-16 $space-24 $space-8;
+    border-bottom: none;
+    p.sub-header {
+      font-size: $fs-card-list-title;
+      color: $spanish-gray;
+      line-height: normal;
+      margin: $space-4 0 0;
+    }
+  }
+
+  .home-external-feed-links-list {
+    padding: $space-16 $space-16 $space-8;
+
+    nb-list {
+      nb-list-item {
+        padding: $space-16 0;
+        margin-bottom: $space-8;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        border: none;
+
+        app-skeleton-text {
+          width: 100%;
+        }
+
+        .home-external-feed-link {
+          margin-right: $space-8;
+
+          .home-external-feed-link-image {
+            min-height: 2.5rem;
+            max-height: 2.5rem;
+            min-width: 2.5rem;
+            max-width: 2.5rem;
```

Comment:
In case of fixed size of height and width, images of the 3 feed elements in home page are coming of different sizes. In the original one, all the images are uniform. Should this change be made ?